### PR TITLE
uname: change sysname flag to kernel-name

### PR DESCRIPTION
**Pull request by tsliang**
The --sysname flag in GNU uname was deprecated in 2002 and replaced by --kernel-name.

### CODE REVIEW

**Comment by Arcterus** [comment]
AFAICT `--sysname` is still supported, it's just hidden in the usage information.  Can you switch the util from `getopts` to `clap` and add `--sysname` as an alias?  That way the behavior should be the same as the GNU version.

**Comment by tsliang** [comment]
Sure, will do. Should I be worried about that failed CI build btw, or does that happen to every PR?

**Comment by Arcterus** [comment]
I don’t think it’s a problem.  The build just errored out because it couldn’t download the compiler (lol).

**Comment by tsliang** [comment]
@Arcterus ok, everything's updated. I added the `--sysname` alias, along with `--release` (another obsolete alias). I also updated the option help list; it's now word-for-word the same with GNU uname.

**Comment by Arcterus** [comment]
Do you mind changing the help menu back to what it was before?  I'm not sure if making it the exact same as the GNU version violates the GPL or whatever GNU licenses its documentation under.  Other than that this looks ready to go!

**Comment by tsliang** [comment]
@Arcterus sorry for the late reply - holidays and all 😃 help menu has been changed to what it was before!

**Comment by Arcterus** [comment]
Can you actually squash the last two?  Just to get rid of the GNU help strings from the commit history.

**Comment by tsliang** [comment]
@Arcterus you got it

**Comment by Arcterus** [comment]
Alright, thanks!  I'll merge this after one of the builds finishes.

**Comment by Arcterus** [comment]
Also I just saw your comment about the holidays.  No worries, I've been busy too 😄

File: src/uname/uname.rs [hunk 1]
```diff
@@ -32,48 +47,76 @@ static HOST_OS: &'static str = "Darwin";
...
+        .usage(&usage[..])
+        .arg(Arg::with_name(OPT_ALL)
+            .short("a")
+            .long(OPT_ALL)
+            .help("Behave as though all of the options -mnrsv were specified."))
+        .arg(Arg::with_name(OPT_KERNELNAME)
+            .short("s")
+            .long(OPT_KERNELNAME)
+            .alias("sysname") // Obsolescent option in GNU uname
+            .help("print the operating system name.")
```
Comment (Arcterus) [inline src/uname/uname.rs]: I think you deleted a `)` by accident when you rebased.

Comment (tsliang) [reply src/uname/uname.rs]: Eep! Sorry about that.

**Comment by Arcterus** [comment]
Alright seems good now.  Thanks!

**Pull request merged**